lava_test_plans/include/fastboot.jinja2

```diff
@@ -140,6 +154,90 @@ reboot_to_fastboot: {{ reboot_to_fastboot }}
             compression: {{OVERLAY_PERF_URL_COMP}}
 {% endif %}
 {% endif %}
+{% endif %}
+{% if partition_boot_a == true %}
+      boot_a:
+        url: {{ BOOT_A_URL }}
+{% if BOOT_A_URL_COMP is defined %}
+        compression: {{ BOOT_A_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_boot_b == true %}
+      boot_b:
+        url: {{ BOOT_B_URL }}
+{% if BOOT_B_URL_COMP is defined %}
+        compression: {{ BOOT_B_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_super == true %}
+      super:
+        url: {{ SUPER_URL }}
+{% if SUPER_URL_COMP is defined %}
+        compression: {{ SUPER_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_system == true %}
+      system:
+        url: {{ SYSTEM_URL }}
+{% if SYSTEM_URL_COMP is defined %}
+        compression: {{ SYSTEM_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_vendor == true %}
+      vendor:
+        url: {{ VENDOR_URL }}
+{% if VENDOR_URL_COMP is defined %}
+        compression: {{ VENDOR_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_vendor_boot == true %}
+      vendor_boot:
+        url: {{ VENDOR_BOOT_URL }}
+{% if VENDOR_BOOT_URL_COMP is defined %}
+        compression: {{ VENDOR_BOOT_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_cache == true %}
+      cache:
+        url: {{ CACHE_URL }}
+{% if CACHE_URL_COMP is defined %}
+        compression: {{ CACHE_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_vbmeta == true %}
+      vbmeta:
+        url: {{ VBMETA_URL }}
+{% if VBMETA_URL_COMP is defined %}
+        compression: {{ VBMETA_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_vbmeta_a == true %}
+      vbmeta_a:
+        url: {{ VBMETA_A_URL }}
+{% if VBMETA_A_URL_COMP is defined %}
+        compression: {{ VBMETA_A_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_vbmeta_b == true %}
+      vbmeta_b:
+        url: {{ VBMETA_B_URL }}
+{% if VBMETA_B_URL_COMP is defined %}
+        compression: {{ VBMETA_B_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_recovery == true %}
+      recovery:
+        url: {{ RECOVERY_URL }}
+{% if RECOVERY_URL_COMP is defined %}
+        compression: {{ RECOVERY_URL_COMP }}
+{% endif %}
+{% endif %}
+{% if partition_userdata == true %}
+      userdata:
```

Comment:
rootfs is one logic concept, while userdata is a real partition, I think it's the place where rootfs is used to set the correct partition it will use, instead of setting here to use the rootfs_label. If for some devices, the system or super partition is used as the rootfs for the lts setup, should the rootfs label be changed to system or super?